packages/now-ruby/test/fixtures/05-sinatra/now.json

```
@@ -1,5 +1,9 @@
 {
   "version": 2,
-  "builds": [{ "src": "index.ru", "use": "@now/ruby" }],
+  "functions": {
+    "api/**/*.ru": {
+      "runtime": "@now/ruby@1.0.1"
+    }
+  },
   "probes": [{ "path": "/", "mustContain": "gem:RANDOMNESS_PLACEHOLDER" }]
```

This probe is wrong since you moved the file.

```suggestion
  "probes": [{ "path": "/api", "mustContain": "gem:RANDOMNESS_PLACEHOLDER" }]
```